things/service_test.go

```diff
@@ -24,10 +24,11 @@ const (
 	token      = "token"
 	token2     = "token2"
 	n          = uint64(10)
+	Prefix     = "fe6b4e92-cc98-425e-b0aa-"
 )
 
 var (
-	thing   = things.Thing{Name: "test"}
+	thingList = [n]things.Thing{}
```

Comment:
Here you can do `things = []things.Thing{{Name: "thing1"},{Name: "thin2"}}`
No need to use entities with external ID for simple Thing creation.

Reply:
The reason is that, a new function has been included by me TestInit() in the original patch.  We have to have Name, ID and Key, that too for 10 Things.  I thought it would be too long to keep all these 10 things with associated Name, ID and Key in the var list.  If required, we can also do the way you suggested.